tiny_dnn/layers/fully_connected_layer.h

```diff
@@ -117,6 +117,12 @@ class fully_connected_layer : public layer {
   /* The layer parameters */
   fully_params params_;
 
+  /* forward op context */
+  OpKernelContext fwd_ctx_;
```

Comment:
OK nice move, but maybe context in a future should also contain shared attributes among all instances of the layer e.g. cuda/ocl kernels, etc